debian/enterprise-control-center/Dockerfile

```diff
@@ -33,7 +33,7 @@ ENV CONTROL_CENTER_CONFIG_DIR=/etc/confluent-${COMPONENT}
 EXPOSE 9021
 
 RUN echo "===> Installing ${COMPONENT}..." \
-    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}-${CONFLUENT_DEB_VERSION} \
+    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}${CONFLUENT_PLATFORM_LABEL}-${CONFLUENT_DEB_VERSION} \
```

Comment:
What does `CONFLUENT_PLATFORM_LABEL` refer to exactly? What form should it take? I see that it can be set to either `CONFLUENT_RPM_LABEL` or `CONFLUENT_DEB_LABEL`, but I can't see what those would be set to.